src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -62,13 +62,19 @@ class ProjectCard extends React.Component {
       ? t('Remove from bookmarks')
       : t('Add to bookmarks');
 
+    let platforms;
+    if (typeof project.platforms !== 'undefined') {
+      platforms = project && project.platforms;
+    } else {
+      platforms = [];
+    }
```

Comment:
no need to change but couldn't this be `const platforms = project && project.platforms || []`?